test_driver/setup/main_test_setup.dart

```diff
@@ -77,11 +80,36 @@ Future<FlutterDriver> setupAndGetDriver({bool isLogin = false}) async {
     } catch (error) {}
   }
   if (!isLogin) {
-    await getAuthentication(driver, providerCoiDebug, emailReal, passwordReal);
+    await getAuthentication(driver, targetProvider, providerEmail, providerPassword);
   }
   return driver;
 }
 
+Future initProviders()  async{
+  final providerName1 = "name1";
```

Comment:
We should probably use something like contact1 and contact2, as name doesn't imply which name.